pkg/lib/operatorstatus/status.go

```diff
@@ -246,38 +260,52 @@ func findOperatorStatusCondition(conditions []configv1.ClusterOperatorStatusCond
 
 // relatedObjects returns RelatedObjects in the ClusterOperator.Status.
 // RelatedObjects are consumed by https://github.com/openshift/must-gather
-func relatedObjects(name, namespace string) []configv1.ObjectReference {
+func relatedObjects(name string, opClient operatorclient.ClientInterface, crClient versioned.Interface) ([]configv1.ObjectReference, error) {
 	var objectReferences []configv1.ObjectReference
+	log.Infof("Adding related objects for %v", name)
+	namespace := clusterOperatorOLM // hard-coded to constant
```

Comment:
This is an incorrect namespace. A default namesspace is `openshift-operator-lifecycle-manage`. The `operator-lifecycle-manage` is the name of the `ClusterOperator` for OLM. You just need to pass `defaultCatalogNamespace` as namespace instead of `catalogNamespace`  for `main.go` for catalog.

Reply:
Ok, ack on the namespace not being the correct openshift one. But I thought we agreed to only query the OLM namespace only and not the marketplace one?